fizzbuzz/index.html

```diff
@@ -0,0 +1,14 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="description" content="">
+    <title>Methods 2022 - Topics</title>
```

Comment:
Maybe also change the title here the Methods 2022 - FizzBuzz instead of Methods 2022 - Topics ?

Reply:
Ill get right on that!